content/integrations/kiali.md

```diff
@@ -0,0 +1,107 @@
+summary: Describes iter8's integrations with Kiali
+---
+
+Kiali is an observability console for Istio with service mesh configuration capabilities. It helps you to understand the structure of your service mesh by inferring the topology, and also provides the health of your mesh. Kiali provides detailed metrics, and a basic Grafana integration is available for advanced queries. Distributed tracing is provided by integrating Jaeger. For Detail Installation and Information about Kiali in general, please reference [Kiali.io](https://kiali.io)
+
+## Enabling the iter8 Console in Kiali
+
+### A. Install Kiali Using Operator
+
+Kiali version 1.18.1 (and above) now provides the capabilities to observe iter8 experiment runtime behavior. To enable the Kiali iter8 extensions, which are disabled by default, you must update the Kiali CR using the Kiali Operator.
+
+**Note**: Currently the iter8 extension for Kiali works only for [iter8 v0.2.1](https://www.iter8.tools), not the current version. To install this version of iter8, see [here](https://iter8.tools/getting-started/installation/kubernetes/).
+
+To check if Kiali operator is installed, use:
+
+```bash
+kubectl get pods -n kiali-operator
+```
+
+To install the Kiali Operator, please follow the steps in [Advanced Install](https://kiali.io/documentation/latest/installation-guide/#_advanced_install_operator_only). And make sure the Kiali CR is created by using command:
+
+```bash
+kubectl get kialis.kiali.io kiali -n kiali-operator
+```
+
+If this is the new installation, you will be asked to choose an authentication strategy (login, anonymous, ldap, openshift, token and openid). Depending on the chosen strategy, the installation may prompt for additional information. Please reference [Kiali Login Options](https://kiali.io/documentation/latest/installation-guide/#_login_options) for details about authentication strategies.
+
+### B. Enable iter8 in the Kiali Operator CR
+
+1. Follow the step [Create or Edit the Kiali CR](https://kiali.io/documentation/latest/installation-guide/#_create_or_edit_the_kiali_cr) or use:
+
+    ```bash
+    kubectl edit kialis.kiali.io kiali -n kiali-operator
+    ```
+
+2. Enable `iter_8` extensions under `Spec`.
+
+    ```
+    # Kiali enabled integration with Iter8 project.
+    # If this extension is enabled, Kiali will communicate with Iter8 controller allowing to manage Experiments and review results.
+    # Additional documentation https://iter8.tools/
+    #    ---
+    #    iter_8:
+    #
+    # Flag to indicate if iter8 extension is enabled in Kiali
+    #      ---
+    #      enabled: false
+    #
+    extensions:
+      iter_8:
+        enabled: true
+    ```
+
+3. Restart the Kiali pods:
+
+    ```bash
+    kubectl -n istio-system delete pod $(kubectl -n istio-system get pod --selector='app=kiali' -o jsonpath='{.items[0].metadata.name}')
+    ```
+
+    You can inspect the pods using `kubectl get pods -n istio-system` to check if Kiali pod has restarted.
```

Comment:
Same, also copyable code block?

Reply:
@Alan-Cha I tried. Perhaps it is a feature of the preview server?  I did get the pictures to work :)